.idea/externalDependencies.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ExternalDependencies">
+    <plugin id="google-java-format" min-version="1.13.0" max-version="1.13.0" />
```

Comment:
Pin the version so that the tool does not auto-upgrade and cause another wave of changes like the one on this PR